ntbs-service/Pages/Notifications/Edit/Items/SocialContextVenue.cshtml.cs

```diff
@@ -163,23 +163,23 @@ public ContentResult OnGetValidateVenueDate(string key, string day, string month
             return ValidationService.ValidateDate<SocialContextVenue>(key, day, month, year);
         }
 
-        // public ContentResult OnGetValidateVenueDates(IEnumerable<Dictionary<string, string>> keyValuePairs)
-        // {
-        //     var propertyValueTuples = new List<Tuple<string, object>>();
-        //     foreach (var keyValuePair in keyValuePairs)
-        //     {
-        //         var formattedDate = new FormattedDate() { Day = keyValuePair["day"], Month = keyValuePair["month"], Year = keyValuePair["year"] };
-        //         if (formattedDate.TryConvertToDateTime(out DateTime? convertedDob))
-        //         {
-        //             propertyValueTuples.Add(new Tuple<string, object>(keyValuePair["key"], convertedDob));
-        //         }
-        //         else
-        //         {
-        //             // should not ever get here as we validate individual dates first before comparing
-        //             return Content(ValidationMessages.ValidDate);
-        //         }
-        //     }
-        //     return ValidationService.ValidateMultipleProperties<SocialContextVenue>(propertyValueTuples);
-        // }
+        public ContentResult OnGetValidateVenueDates(IEnumerable<Dictionary<string, string>> keyValuePairs)
+        {
+            var propertyValueTuples = new List<Tuple<string, object>>();

```

Comment:
Would using value tuples be nicer here? Then we can give them a bit more meaning too, with `List<(string key, object property)>`?

Reply:
ohh I hadn't seen them before, they're great!